CRM/Report/Form/Contribute/Summary.php

```diff
@@ -827,6 +811,7 @@ public function alterDisplay(&$rows) {
       }
 
       $entryFound = $this->alterDisplayAddressFields($row, $rows, $rowNum, 'contribute/detail', 'List all contribution(s) for this ') ? TRUE : $entryFound;
+      $entryFound = $this->alterDisplayContactFields($row, $rows, $rowNum, 'contribute/detail') ? TRUE : $entryFound;
```

Comment:
Thanks Yashi, I altered it so the signatures for the 2 functions match & updated the PR. I assigned the JIRA back to Jitendra to re-QA. I didn't replicate all the e-notices Jitendra hit so will await feedback on that.